website/styles.css

```
@@ -143,8 +109,8 @@ p  { margin: 0 0 1rem; color: var(--c-ink-dim); }
   font-weight: 800; letter-spacing: .02em; font-size: 1.05rem;
   color: var(--c-ink);
 }
-.brand-mark { color: var(--c-signature); font-size: 1.1em; }  /* DARKXSIDE ✦ */
-.brand-word { font-family: var(--f-display); letter-spacing: .02em; }
+.brand-mark { color: var(--c-accent); font-size: 1.1em; }
+.brand-word { }
```

Stylelint (17.14.0)

[error] Empty block (block-no-empty)

(block-no-empty)